### PR TITLE
Identity Recovery Kits

### PR DESCRIPTION
**Pull request by aince42**
# Readiness checklist

- [ ] I added/updated tests.
- [x] I ensured that the PR title is good enough for the changelog.
- [x] I labeled the PR.
- [x] I self-reviewed the PR.

### CODE REVIEW

File: apps/enmeshed/lib/account/home/widgets/complete_profile_container.dart
Comment (jkoenig134) [inline apps/enmeshed/lib/account/home/widgets/complete_profile_container.dart:1]: ln 82 must be removed (the alwaysChecked entry)

Comment (jkoenig134) [reply apps/enmeshed/lib/account/home/widgets/complete_profile_container.dart:1]: if you don't do that we will have `5 von 4 vervollständigt`

Comment (jkoenig134) [reply apps/enmeshed/lib/account/home/widgets/complete_profile_container.dart:1]: Also `countCompleted:` is just not updated with the value from the recovery kits.

File: apps/enmeshed/assets/svg/files.svg
Comment (jkoenig134) [inline apps/enmeshed/assets/svg/files.svg:1]: please help me - why is this called `files.svg`?

File: apps/enmeshed/lib/core/widgets/instructions_screen.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ import 'package:vector_graphics/vector_graphics.dart';
 
 import '/core/core.dart';
 
-enum InstructionsType { addContact, loadProfile }
+enum InstructionsType { addContact, loadProfile, createKit }
```
Comment (jkoenig134) [inline apps/enmeshed/lib/core/widgets/instructions_screen.dart:9]: what the heck is a kit? 🧐

File: apps/enmeshed/lib/core/modals/create_recovery_kit.dart [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+              ),
+            ),
+          ],
+        ),
+        if (widget.password == null || _pdf == null) ModalLoadingOverlay(text: context.l10n.identityRecovery_generatingInProgress, isDialog: false),
+      ],
+    );
+  }
+
+  // Download is not working on iOS simulator (but works on real device)
```
Comment (jkoenig134) [inline apps/enmeshed/lib/core/modals/create_recovery_kit.dart:250]: ?

Comment (jkoenig134) [reply apps/enmeshed/lib/core/modals/create_recovery_kit.dart:250]: should have popped up in your self review.

Comment (aince42) [reply apps/enmeshed/lib/core/modals/create_recovery_kit.dart:250]: I noticed it, but why should that bother us if it works on real devices without any problems?

Comment (aince42) [reply apps/enmeshed/lib/core/modals/create_recovery_kit.dart:250]: Ah sorry

**Comment by jkoenig134** [comment]
Will be implemented in #367

**Pull request closed without merging**